src/main/kotlin/Main.kt

```diff
@@ -1,36 +1,54 @@
 import java.lang.Math.abs
-import java.lang.Math.random
 import java.util.*
 
 fun main(args: Array<String>) {
-    val r = Random(System.nanoTime()).nextInt(-100, 100)
-    val a = 13
-    val n = IntArray(13)
+    
+    val array = makeArrae()
+    val maxAbsIndex = findMaxAbsIndexInArray(array)
+
+    printMaxAbsElementInArray(array, maxAbsIndex)
+}
+
+fun randomNumber(): Int {
+    return Random(System.nanoTime()).nextInt(-100, 100)
+}
+
+fun printMaxAbsElementInArray(array: IntArray, MaxAbsInex: Int) {
+    for ( item in array ) {
+        print("$item, ")
+    }
+    println()
+    print("Максимальный по модулю $MaxAbsInex элемент равен ${array[MaxAbsInex]}")
+}
+fun findMaxAbsIndexInArray(array: IntArray): Int {
```

Comment:
Между функциями и методами лучше делать один отступ
```suggestion
}

fun findMaxAbsIndexInArray(array: IntArray): Int {
```